docs/tutorials/default-environment.md

```diff
@@ -0,0 +1,194 @@
+the `default` enviroment is simply an environment in your home directory.
+Since you're unlikely to do development in your home directory we treat this
+environment specially.
+
+In some cases Flox will prompt to set up your `default` environment for you.
+To create the `default` environment yourself,
+simply navigate to your home directory and run [`flox init`][init].
+
+```bash
+$ cd ~
+$ flox init
+```
+
+Once the environment has been created,
+you'll want to configure your shell to activate the environment with every new
+shell.
+This can be done as part of the automatic setup,
+or you can add a single line to your shell's RC file:
+
+=== "Bash"
+
+    Add the following line to the very end of your `.bashrc` file:
+
+    ```bash
+    eval "$(flox activate -m run)"
+    ```
+
+=== "Zsh"
+
+    Add the following line to the very end of your `.zshrc` file:
+
+    ```bash
+    eval "$(flox activate -m run)"
+    ```
+
+=== "Fish"
+
+    Add the following line to the very end of your `config.fish` file:
+
+    ```fish
+    flox activate -m run | source
+    ```
+
+=== "Tcsh"
+
+    Add the following line to the very end of your `.tcshrc` file:
+
+    ```bash
+    eval "`flox activate -m run`"
+    ```
+
+---
+
+Once you've added that line to your shell,
+you'll need to restart your shell (or open a new one) for the changes to
+take effect.
+
+## Taking it for a spin
+
+Now let's test out your new `default` environment.
+If you're in an arbitrary directory and `apt install hello` you would expect
+it to be available no matter what directory you're in.
+Let's do the same with Flox.
+
+Let's create a new directory that we know doesn't have an environment in it.
+
+```bash
+# Create and enter a new temporary directory
+$ cd $(mktemp -d)
+```
+
+Now we'll install a package and see that it gets installed to the `default`
+environment,
+like you would expect from your system's package manager:
+
+```bash
+$ flox install hello
+✅ 'hello' installed to environment 'default'
+```
+
+It worked (though you shouldn't be surprised, Flox is awesome)!
+
+## Customization
+
+Depending on when you created your default environment
+(the default was changed recently),
+you may also see `flox [default]` as part of your prompt for every new shell.
+You can configure that with a single command:
+
+=== "Do show the Flox prompt"
+
+    ```bash
+    $ flox config --set-bool hide_default_prompt false
+    ```
+
+=== "Don't show the Flox prompt"
+
+    ```bash
+    $ flox config --set-bool hide_default_prompt true
+    ```
+
+---
+
+## Sharing
+
+Since the `default` environment is "just" another Flox environment,
+it's possible to push this environment and share it between machines.
+
+From the machine with your `default` environment set up the way you like it,
+run the [`flox push`][push] command:
+
+```bash
+$ cd ~
+$ flox push
+```
+
+You may need to authenticate with FloxHub first,
+but once that completes you're now able to share this environment with another
```

Comment:
**nitpick:** When used as a verb, "complete" needs to be transitive. How about this?
```suggestion
but when that's done, you're now able to share this environment with another
```